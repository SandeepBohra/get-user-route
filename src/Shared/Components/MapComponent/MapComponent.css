#map {
    margin: 10px;
    top: 0%;
    left: 25%;
    position: absolute;
    width: 70%;
    height: 90%;
}