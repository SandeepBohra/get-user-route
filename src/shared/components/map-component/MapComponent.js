import React,{ Component } from 'react';
import './MapComponent.css'

class MapComponent extends Component {
    render() {
        return(
            <div className="MapComponent">
                <div className="map-container">
                    <div id='map' ref={this.props.fieldRef}></div>
                </div>
            </div>
        )
    }
}

export default MapComponent;