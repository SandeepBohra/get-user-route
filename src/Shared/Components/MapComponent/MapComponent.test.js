import React from 'react';
import renderer from 'react-test-renderer';
import MapComponent from './MapComponent';

describe('<MapComponent /> component', () => {
  
  it('should match the snapshot', () => {
    const output = renderer.create(<MapComponent />).toJSON();
    expect(output).toMatchSnapshot();
  })
})