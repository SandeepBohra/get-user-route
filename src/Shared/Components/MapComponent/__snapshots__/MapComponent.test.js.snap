// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MapComponent /> component should match the snapshot 1`] = `
<div
  className="MapComponent"
>
  <div
    className="map-container"
  >
    <div
      id="map"
    />
  </div>
</div>
`;
